modules/lily-compiler/src/main/java/io/github/tomboyo/lily/compiler/icg/OasComponentsToAst.java

```diff
@@ -74,33 +79,33 @@ public static Stream<Ast> evaluate(
    * references which evaluate from this array schema definition.
    */
   private static Stream<Ast> evaluateArray(
-      PackageName basePackage,
-      Pair<AstReference, Stream<Ast>> refAndAst,
-      SimpleName componentName) {
-    var ast = refAndAst.right().collect(toList());
+      PackageName basePackage, Pair<Fqn, Stream<Ast>> fqnAndAst, SimpleName componentName) {
+    var ast = fqnAndAst.right().collect(toList());
     // 1. Create a mapping of FQNs that need to move => their updated package names.
```

Comment:
follow-up ticket to use new variant of OasSchemaToAst.evaluate that suppoorts a genPackage target in addition to a base package.